File: src/Systems/Swift/patterns/Command.swift
enum CommandExample {
    static func run() -> Bool {
        let commands: [(Int) -> Int] = [{ $0 + 50 }, { $0 - 20 }]
        let balance = commands.reduce(100) { value, command in command(value) }
        return balance == 130 && commands[1](150) == 130
    }
}

File: src/Systems/C/patterns/command.c
#include <stdbool.h>
static int a(int x){return x+50;}static int s(int x){return x-20;}bool run(void){int b=100;int(*o[2])(int)={a,s};for(int i=0;i<2;i++)b=o[i](b);return b==130;}
